src/remarkable-ui/charts/tables/HeatMap/HeatMap.types.ts

```diff
@@ -0,0 +1,34 @@
+export type HeatMapPropsDimension<T> = {
+  key: Extract<keyof T, string>;
+  label: string;
+  format?: (value: string) => string;
+};
+
+export type HeatMapPropsMeasure<T> = {
+  key: Extract<keyof T, string>;
+  label: string;
+  format?: (value: number) => string;
+};
+
+export type HeatMapPropsThreshold = string | number | `${number}%` | undefined;
```

Comment:
```suggestion
export type HeatMapPropsThreshold =  number | `${number}%` | undefined;
```
If i'm not wrong this can be updated to be without string.